### DISCUSSION TITLE
Libre compatibility

### DISCUSSION THREAD

**Discussion by GLoren1** [NightscoutFoundation/xDrip#3568] [Q&A]
I've been reading about the compatibility of U.S. version of Freestyle Libre (2 and 3), but I don't understand fully whether either of these are compatible (with or without patching).  Could you help by clarifying? Thanks

**Comment by PhilTU** [comment]
No  L2/L3 US sensor will work directly with xDrip as I understand it.

Juggluco.nl has compatibility with those sensors. It can (should ?) be used on its own, but can feed data into xDrip+ by emulating the Libre 2 patched app data source.
